runtime/src/lib.rs

```diff
@@ -139,7 +139,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     //   `spec_version`, and `authoring_version` are the same between Wasm and native.
     // This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
     //   the compatible custom types.
-    spec_version: 191,
+    spec_version: 192,
```

Comment:
curious if the code change just to pass CI. no need to update runtime spec version.

Reply:
If I don't do this then one checker failed. Also this action is mentioned in the [contribution instruction](https://github.com/opentensor/subtensor/blob/main/CONTRIBUTING.md).